example/reset-all.sh

```diff
@@ -12,6 +12,6 @@ set -e
 eval $compose run --rm dmss reset-app
 eval $compose run --rm job-api dm -u http://dmss:5000 reset ../app
 dm reset app
-dm import-plugin-blueprints node_modules/@development-framework/dm-core-plugins
+dm import-plugin-blueprints ../node_modules/@development-framework/dm-core-plugins
```

Comment:
Don't think this change is necessary since the old script worked if following the README (which specifies that user should run the reset-all script from the example folder.

Reply:
Yeah, but it stopped working when we updated dm-core-plugins version. yarn workspaces will and should install dm-core-plugins in node_modules in root folder so there was no dm-core-plugins modules in example project folder